src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const itemsList = list.querySelectorAll('li');
+
+itemsList.forEach(li => {
+  if (li) {
```

Comment:
The 'if (li)' condition is not necessary since forEach will only iterate over existing elements in the 'itemsList'. You can safely remove this condition.

Reply:
ty

Reply:
all fixed :)